io.itsterminal.WebPConverter.json

```diff
@@ -0,0 +1,35 @@
+{
+  "app-id": "io.itsterminal.WebPConverter",
```

Comment:
how do we verify you own it? The webpage says 403

Reply:
I don't see any footer

![image](https://github.com/user-attachments/assets/ec69782a-d472-44c9-8fa9-77ba394b66ab)

Reply:
you can check again now.